ecl/eclagent/eclagent.cpp

```diff
@@ -3307,6 +3268,18 @@ int myhook(int alloctype, void *, size_t nSize, int p1, long allocSeq, const uns
 }
 #endif
 //--------------------------------------------------------------
+
+void usage()
+{
+    printf("USAGE: eclagent --wuid=wuid options\n"
+           "       eclagent PROCESS=dllname options\n"
```

Comment:
where is this option implemented?

Reply:
It isn't, I suspect. The usage functions(s) are very inconsistent and out of date. I fixed some issues in them but looks like I didn't get all of them (this is old code that was moved)